src/dra/47BB8.c

```diff
@@ -70,17 +70,17 @@ s32 func_800E7E08(u32 arg0) {
         } while (i < 0x10);
         break;
     case 19:
-        LoadTPage((PixPattern*)0x80280000, 0, 0, 0x2C0, 0x100, 0x100, 0x80);
-        LoadTPage((PixPattern*)0x80284000, 0, 0, 0x2C0, 0x180, 0x80, 0x80);
+        LoadTPage((u8*)0x80280000, 0, 0, 0x2C0, 0x100, 0x100, 0x80);
```

Comment:
The SDK manual says LoadTPage takes a u_long ptr in the first argument. Should we change the prototype in libgpu.h and respect the original castings?